src/listener.rs

```diff
@@ -22,6 +22,7 @@ fn handle_new_conn(new_conn: quinn::NewConnection) {
     // send in their certificate - have a delay and then log and forget them
     let q_conn = new_conn.connection;
     let incoming_streams = new_conn.incoming;
+    current_thread::spawn(new_conn.driver.map_err(|_| ()));
```

Comment:
Same as with the `connect.rs` review above:
> In case of error I would like to destroy the connection object to prevent leaks and limbo states. I don't know what this error means but i guess dropping it is the safest way to go ?